nnvm/tests/python/frontend/tensorflow/test_forward.py

```diff
@@ -877,10 +877,91 @@ def test_forward_transpose():
     _test_forward_transpose((2, 3, 4), (0, 1, 2))
     _test_forward_transpose((2, 3, 4, 5), (3, 0, 1, 2))
 
+#######################################################################
+# Less
+# ----
+def _test_forward_less(data):
```

Comment:
I think these test(less, less_equal, greater, greater_equal) are very similar. Could you put these in one as with [this](https://github.com/dmlc/tvm/blob/master/nnvm/tests/python/frontend/tensorflow/test_forward.py#L333-L349)?